js/app/Spinner.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var $ = require( 'jquery' );
+
+var Spinner = function( $node ) {
+	this._$node = $node;
+};
+
+$.extend( Spinner.prototype, {
+	/**
+	 * @type {jQuery}
+	 */
+	_$node: null,
+
+	/**
+	 * Adds a spinner with `size` px height to a node.
+	 *
+	 * @param {int} size
```

Comment:
Oops, I forgot that I decided to set the size in the CSS instead halfway through the implementation. Well spotted! Will change that in a bit.